hermes-common/src/main/java/pl/allegro/tech/hermes/common/message/wrapper/AvroMessageContentUnwrapperResult.java

```diff
@@ -0,0 +1,33 @@
+package pl.allegro.tech.hermes.common.message.wrapper;
+
+class AvroMessageContentUnwrapperResult {
```

Comment:
I'm considering sacrificing this class (together with some readability) for the sake of performance and returning simple null/not null result.

Reply:
Yeah, that was exactly what I was thinking about in that comment

Reply:
I did a review, but forgot to click submit PR, sorry)